server/src/weaverbird/backends/pypika_translator/translators/snowflake.py

```diff
@@ -34,6 +35,23 @@ class SnowflakeTranslator(SQLTranslator):
     REGEXP_OP = RegexOp.REGEXP
     TO_DATE_OP = ToDateOp.TO_TIMESTAMP_NTZ
 
+    @classmethod
+    def sanitize_column_name(cls, col: str) -> str:
```

Comment:
Don't we want this to be for all backends? In which case it could be a pure util function and not a method

Reply:
mhhh... am not sure we have to propagate this...
i mean at this time, the only place we found this issue was on snowflake aggregations...so... idk..

Reply:
I may be wrong but at this day...  postgresql, bigquery, athena or redshift... seems not having this issue...